pkg/codegen/testing/test/testdata/plain-object-defaults/dotnet/Inputs/LayeredTypeArgs.cs

```diff
@@ -28,7 +28,7 @@ public sealed class LayeredTypeArgs : Pulumi.ResourceArgs
         /// Test how plain types interact
         /// </summary>
         [Input("plainOther")]
-        public Inputs.HelmReleaseSettings? PlainOther { get; set; }
+        public Inputs.HelmReleaseSettingsArgs? PlainOther { get; set; }
```

Comment:
I'm a little confused why this change to `go/gen.go` and `schema/schema.go` resulted in renaming a lot of other types. Is this going to require a major version bump for our other packages due to renames?

Reply:
Because we change how `plain` types are generated back to the pre https://github.com/pulumi/pulumi/pull/9430 algorithm. As it stands, pulumi/pulumi currently exhibits a breaking change which this PR undoes. The result is no breaking change between our last release (pre https://github.com/pulumi/pulumi/pull/9430) and our next release (post #9488).